src/utils.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+
+export const PLACEHOLDER_REGEX = /xxx(\S)xxx/gm
```

Comment:
This is the only thing I'm concerned about because patterns with a `g` modifier are stateful. I'm not 100% it matters for us here but I want to be sure.

See here (comments as well)
https://ponyfoo.com/articles/stateful-regular-expressions